veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,253 @@
+# VEP #21: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Kubevirt users can benefit from seamless live migration of VMs without disconnection of TCP sessions, while using `passt`.
+This functionality has recently been introduced by `passt` alongside support of `vhost-user` protocol, which increases network throughput. 
+This proposal outlines the integration plan to be implemented in Kubevirt. 
+
+## Motivation
+
+Currently, KubeVirt does not support all Primary Network requirements expected by users:
+- Seamless network connectivity on migration
+- Observability/Service Mesh support
+- High throughput (on par with tap)
+
+Bridge binding does not support observability or service meshes, Masquerade hides the real IP from the user, and passt currently lacks seamless migration and tap-level performance.
+These gaps have been addressed in the recently released version of passt. 
+
+## Future potential enhancement
+
+Support for all IP network protocols, in addition to the currently supported TCP/UDP/ICMP.
+This capability is planned to be implemented in future releases of passt, and it too is expected to require adaptation in Kubevirt.
+
+## Goals
+
+- Seamless live migration of TCP connections without disconnection for Primary Network.
+- Support for observability tools and service mesh technologies.
+- Enhanced IPv6 support.
+- High network throughput, on par with Masquerade and Bridge bindings.
+
+## Non Goals
+
+- Supporting network protocols other than TCP/UDP/ICMP (ARP/DNS).
```

Comment:
TCP/UDP/ICMPv6 (ICMPv6 is a different protocol compared to ICMP)

As it's a non-goal, the support status of ARP and DNS becomes a bit unclear like this. Maybe "(currently supported protocols also include ARP)". I'd skip DNS, it's on top of either UDP or TCP.